### PR TITLE
VideoPlayer: Create submenu to set sizing mode

### PR DESCRIPTION
**Pull request by timre13**
Create submenu to set sizing mode, also add icon to the sizing mode cycling button.

### CODE REVIEW

File: Userland/Applications/VideoPlayer/VideoPlayerWidget.cpp [hunk 1]
```diff
@@ -83,6 +84,26 @@ ErrorOr<void> VideoPlayerWidget::setup_interface()
     find_descendant_of_type_named<GUI::Button>("sizing")->set_action(*m_cycle_sizing_modes_action);
     find_descendant_of_type_named<GUI::Button>("fullscreen")->set_action(*m_toggle_fullscreen_action);
 
+    m_size_fit_action = GUI::Action::create_checkable("&Fit", [&](auto&) {
+        m_video_display->set_sizing_mode(VideoSizingMode::Fit);
+        m_video_display->update();
```
Comment (AtkinsSJ) [inline Userland/Applications/VideoPlayer/VideoPlayerWidget.cpp:89]: If we have to update the VideoFrameWidget every time we set the sizing mode on it, maybe `set_sizing_mode()` should do that intsead. (After checking that it's actually different from the current mode.)

Comment (timre13) [reply Userland/Applications/VideoPlayer/VideoPlayerWidget.cpp:89]: How do I update the commit? Should I amend and force-push it? (I'm still new)

Comment (konradekk) [reply Userland/Applications/VideoPlayer/VideoPlayerWidget.cpp:89]: yup!

**Review by AtkinsSJ — APPROVED** [review]
:+1:

**Pull request merged**